test/deploy/rackhd_ha_resource_install.py

```diff
@@ -166,15 +138,16 @@ def set_rabbitmq_cluster_policy(self, vmnum, rabbit_rsc_list):
         rabbitmq_policy = open('rabbitmq.bash', 'w')
         rabbitmq_policy.write(rendered)
         rabbitmq_policy.close()
-        # copy file to ORA
-        fit_common.scp_file_to_host('rabbitmq.bash', vmnum=node)
-        fit_common.remote_shell("chmod 777 rabbitmq.bash", vmnum=node)
-        self.assertEqual(fit_common.remote_shell("./rabbitmq.bash", vmnum=node)['exitcode'],
-                         0, "Rabbitmq mirrored queue policy failure.")
 
     def test02_install_rabbitmq_resource(self):
```

Comment:
can this 'test02_install_rabbitmq_resource' run if test01 fails?  is so, great. 
if not, add the nosedep dependency in to skip the test.